.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,40 +1,78 @@
 ---
-name: "🐛 Bug Report"
-about: Report a reproducible bug or regression in React Native.
+name: "🐛 React Native for macOS Bug Report"
+about: Report a reproducible bug or regression in React Native for macOS.
 title: ''
 labels: 'Bug'
 
 ---
 
+### Environment
+
 <!--
-  Please provide a clear and concise description of what the bug is.
-  Include screenshots if needed.
-  Please test using the latest React Native release to make sure your issue has not already been fixed: http://facebook.github.io/react-native/docs/upgrading.html
+  Please fill in all the relevant fields by running these commands in terminal.
 -->
+1. `react-native -v`:
+3. `npm ls react-native-macos`:
+4. `node -v`:
+5. `npm -v`:
+6. `yarn --version`<!-- (if you use Yarn) -->:
+7. `xcodebuild -version`:
+
+### Issue
+
+### Steps to Reproduce
 
-React Native version:
 <!--
-  Run `react-native info` in your terminal and copy the results here.
+  How would you describe your issue to someone who doesn’t know you or your project?
```

Comment:
Should we put something about bonus points if they can repro it already in RNTester or make a branch that repros it in RNTester?